wscript

```diff
@@ -483,6 +483,13 @@ def configure(conf):
         defines=['_GNU_SOURCE'],
         mandatory=False)
 
+    # Check for backtrace support
+    conf.check(
+        compiler='c',
```

Comment:
Why compiler='c'? Waf should be able to guess that. See https://github.com/waf-project/waf/blob/master/waflib/Tools/c_config.py#L472